app/01net_lists/main.c

```diff
@@ -0,0 +1,49 @@
+/**
+ * @file
+ * @ingroup     net_lists
+ *
+ * @brief       Example on how to use lists modules
```

Comment:
I would not bother adding these to all files. It's heavy. It was useful when source code was archived on paper; now, with GitHub, we konw exacty who did what

Reply:
Agree, was following DotBot convention, happy to drop.